attack.go

```diff
@@ -111,10 +113,20 @@ func attack(opts *attackOpts) (err error) {
 		src = files[opts.targetsf]
 		hdr = opts.headers.Header
 	)
-	if opts.lazy {
-		tr = vegeta.NewLazyTargeter(src, body, hdr)
-	} else if tr, err = vegeta.NewEagerTargeter(src, body, hdr); err != nil {
-		return err
+
+	switch opts.format {
+	case "json":
```

Comment:
string `json` has 3 occurrences, make it a constant

Reply:
Agree, should be constants.

Reply:
OK, I'll make some constants for these values.